pep_sphinx_extensions/pep_theme/static/style.css

```diff
@@ -248,20 +245,20 @@ dl.footnote {
     line-height: 1.875;
     width: 100%;
     border-top: 1px solid var(--colour-background-accent);
-    border-bottom: 1px solid var(--colour-background-accent);
 }
 dl.rfc2822 > dt, dl.rfc2822 > dd,
 dl.footnote > dt, dl.footnote > dd {
     padding: .25rem .5rem .2rem;
-}
-dl.rfc2822 > dt:nth-of-type(even), dl.rfc2822 > dd:nth-of-type(even),
-dl.footnote > dt:nth-of-type(even), dl.footnote > dd:nth-of-type(even) {
-    background-color: var(--colour-background-accent);
+    border-bottom: 1px solid var(--colour-background-accent);
```

Comment:
```suggestion
    border-bottom: 1px solid var(--colour-accent-strong);
```

The horizontal rules substituted for the alternating colors for the PEP headers and footnotes are very low contrast, which combined with being very thin is such that even I can barely see them, well below even the lowest minimum WCAG accessibility standards.

![image](https://user-images.githubusercontent.com/17051931/158074741-cb07cab0-d461-4a72-a6ca-050b9b210bbd.png)


This is because they use `--colour-background-accent`, rather than the (much more semantically appropriate) ``--colour-rule-strong``, which is specifically intended for rules like this. Switching to that instead is much more easily visible at 4.3:! and passes WCAG AA:

![image](https://user-images.githubusercontent.com/17051931/158074664-74be41b5-6207-4f35-8a6e-eb7920833a0a.png)

Reply:
I don't care, but I do find it weird to be applying rules of text readability contrast to... borders? :)

Reply:
I didn't mind enough to argue! 

A